utilities/src/with_std/metrics.rs

```diff
@@ -624,15 +624,15 @@ mod test {
 				{
 					let mut metrics = CeremonyMetrics::new("Chain1", "Keygen");
 					metrics.bad_message.inc(&["AA"]);
-					metrics.ceremony_duration.observe(999);
+					metrics.ceremony_duration.observe(Duration::new(999, 0));
 					metrics.missing_messages.set(&["stage1",], 5);
 					metrics.processed_messages.inc();
 					metrics.processed_messages.inc();
 					metrics.stage_completing.inc(&["stage1"]);
 					metrics.stage_completing.inc(&["stage1"]);
 					metrics.stage_completing.inc(&["stage2"]);
-					metrics.stage_duration.observe(&["stage1", "receiving"], 780);
-					metrics.stage_duration.observe(&["stage1", "processing"], 78);
+					metrics.stage_duration.observe(&["stage1", "receiving"], Duration::new(780, 0));
+					metrics.stage_duration.observe(&["stage1", "processing"], Duration::new(78, 0));
 					metrics.stage_failing.inc(&["stage3", "NotEnoughMessages"]);
 
 					//This request does nothing, the ceremony is still ongoning so there is no deletion
```

Comment:
Below can we update: "# HELP ceremony_duration Measure the duration of a ceremony in ms" to seconds.

Reply:
And the metric describes of course